docs/FHSS_SHAPING.md

```diff
@@ -0,0 +1,43 @@
+# mLRS Documentation: FHSS Shaping #
+
+([back to main page](../README.md))
+
+mLRS uses frequency hopping. 
+
+The number of frequencies which are used in the hopping depend on the RF band and the selected mode. For instance, in the 2.4 GHz band and for the 50 Hz mode, the FHSS sequence consists of 24 different frequencies. The frequency values and their sequence however are determined by the bind phrase. This allows for one to operate several mLRS systems in the same RF band with minimized mutual interference, by choosing different bind phrases for each system.
+
+In addition to this "bind phrase" mechanism for affecting the frequencies used in the FHSS sequence, mLRS also provides the "except" and "ortho" mechanisms.
```

Comment:
Perhaps:
'In addition to the "bind phrase" mechanism, mLRS also provides the "except" and "ortho" mechanisms for altering the frequencies used in the FHSS sequence

Reply:
THX! done.